addon/serializers/application.js

```diff
@@ -86,7 +86,7 @@ export default DS.RESTSerializer.extend({
         // the links property so the adapter can async call the
         // relationship.
         // The adapter findHasMany has been overridden to make use of this.
-        if(options.relation) {
```

Comment:
Ah my bad. See my comment below about that. Not sure what should be happening with those two options.